cloud/terraform/aws/backup/modules/backup-vault/outputs.tf

```diff
@@ -0,0 +1,6 @@
+output "backup_service_assumed_role_arn" {
```

Comment:
Named `outputs`, `providers`, etc, diverging from current pattern of using singular. Because this is by conventions since the files contain multiple outputs, providers, etc. Currently the code isn't consistent and uses `variables.tf` in the plural only. @rotty3000 , should we update this?

Reply:
Discussed via call. I can send another PR to fix this.